tools/platforms/configs/rbe_default/cc/cc_toolchain_config.bzl

```diff
@@ -1238,7 +1238,7 @@ def _impl(ctx):
         flag_sets = [
             flag_set(
                 actions = [ACTION_NAMES.c_compile, ACTION_NAMES.cpp_compile],
-                flag_groups = [flag_group(flags = ["-Werror"])],
```

Comment:
This is autogenerated and will just be overwritten.

Reply:
Reverted.